### PR TITLE
Add documentation for Wallabag iOS app setup

### PR DESCRIPTION
**Pull request by junderhill**
Add documentation for setup of iOS app against Wallabag instance in response to wallabag/ios-app#7

### CODE REVIEW

File: en/apps/ios.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+# iOS application
+
+## Purpose of this document
+
+This document describes how you can setup your iOS application to
+work with your wallabag instance. There is no difference in this
+procedure for wallabag v1 or v2.
```
Comment (bourvill) [inline en/apps/ios.md]: Wallabag v1 is not supported

Comment (junderhill) [reply en/apps/ios.md]: Apologies - I used the Android docs as a template. I'll change this

Comment (bourvill) [reply en/apps/ios.md]: No problem, nothing serious. thank you for your work

Comment (junderhill) [reply en/apps/ios.md]: Commit 996b843 should have the change - shown below

**Comment by j0k3r** [comment]
I trust @bourvill.
Thanks @junderhill!

**Pull request merged**